native/c/zjbm.c

```diff
@@ -36,11 +39,36 @@ typedef struct
   unsigned char buffer[SYMBOL_ENTRIES * 16];
 } JSYMBOLO;
 
+typedef struct psa PSA;
+typedef struct cvt CVT;
+typedef struct jesct JESCT;
+
+static void get_ssibssnm(SSIB *ssib)
+{
+  PSA *psa_a = (PSA *)0;
+  CVT *cvt_a = psa_a->flccvt;
+  JESCT *jesct_a = cvt_a->cvtjesct;
+
+  if (NULL != jesct_a->jespjesn)
```

Comment:
I dont think this will ever be `NULL` on a system, but if it is, I think we should return an error instead of defaulting to `JES2`